src/Wordpress/CustomPostType/Item.php

```diff
@@ -22,10 +23,33 @@ public function initHooks() {
 		// Listing of locations for item
 		add_shortcode( 'cb_locations', array( \CommonsBooking\View\Location::class, 'shortcode' ) );
 
+		//Add filter to backend list view
+		add_action( 'restrict_manage_posts', array( self::class, 'addAdminCategoryFilter' ) );
+
 		// Filter only for current user allowed posts
 		add_action( 'pre_get_posts', array( $this, 'filterAdminList' ) );
 	}
 
+	/**
+	 * Adds Category filter to backend list view
+	 * 
+	 */
+	public static function addAdminCategoryFilter() {
```

Comment:
@hansmorb : Wenn ich es richtig sehe ist die Funktion für Items und Locations gleich. Von daher würde ich das eher in die CustomPostType Klasse verschieben, dann ist das nicht doppelt.

Reply:
Gute Idee!